client/reader/stream/x-post.jsx

```diff
@@ -80,9 +80,10 @@ class CrossPost extends PureComponent {
 
 	handleClick = ( event ) => {
 		if ( event.button > 0 || event.metaKey || event.controlKey || event.shiftKey || event.altKey ) {
-			return;
+			window.open( this.props.post.URL, '_blank', 'noopener noreferrer' );
```

Comment:
I see the benefit there, but my wily old developer brain wants the HTML to be usable without Javascript ;) I don't think we should reproduce default behaviour.

Unintended side effect of a `<a>` element without a `href` I just learnt about: it won't be in the tabbing order when navigating with a keyboard.

cc @blowery for casting vote